### PR TITLE
In This feature i try to implement the function of the calculator

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
- [ ] Copied the files calculate.js and operate.js into a logic/ directory in my project.
- [ ] IImplemented the event handlers by using the math logic from calculate.js and operate.js.

### CODE REVIEW

File: src/components/Layout/display.value.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Display extends React.Component {
+  // eslint-disable-next-line no-useless-constructor
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // eslint-disable-next-line react/prop-types
```
Comment (hiromataba) [inline src/components/Layout/display.value.js]: - [ ] Linting is important to reduce errors and improve the overall quality of your code. Using lint tools can help you accelerate development and reduce costs by finding errors earlier. so please do not disable them since they are meant to help you :wink:

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Dagic-zewdu :wave: :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_sports: 
- Your app is running correctly :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please update your README file by removing some script in the `Available script` section generated by `CRA` to clean up up your README file and make it more professional 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Layout/display.value.js (hunk 1 above)
Comment (Dagic-zewdu) [reply src/components/Layout/display.value.js]: `// eslint-disable-next-line react/prefer-stateless-function`
This disabling is so important because we are using  class-based component

File: .eslintrc.json [hunk 2]
```diff
@@ -21,5 +21,5 @@
     "no-shadow": "off",
     "react/prop-types": "off"
   },
-  "ignorePatterns": ["dist/", "build/", "docs/"]
+  "ignorePatterns": ["dist/", "build/", "docs/","./src/Logic/calculate.js"]
 }
```
Comment (Gambit142) [inline .eslintrc.json]: - [ ] You did well removing most eslint disabled comments but a general rule is for you not to make custom changes to linter config files. I am sorry but I have to make the hard decision to invalidate this pull request. Kindly undo the changes and submit them again. 👍

**Review by Gambit142 — COMMENTED** [review]
Hi @Dagic-zewdu 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you made custom changes to the Eslint linter config. Kindly undo the change you made. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json (hunk 2 above)
Comment (Dagic-zewdu) [reply .eslintrc.json]: dist,docs folder and the build folder should be ignored but not `"./src/Logic/calculate.js"`

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Dagic-zewdu 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
